config/locales/en.yml

```diff
@@ -4,7 +4,7 @@ en:
       frequency:
         daily: You’ll get one email a day
```

Comment:
In the design, there is a full stop after the word `day` does this need to be here too?

Reply:
That's covered here: https://github.com/alphagov/email-alert-api/pull/1478/files#diff-05a846651f6792b461f7abfa940657266c0d62655a4dbd40d6ee2f2b33527d59R39.